src/core/Object3D.js

```diff
@@ -620,6 +621,11 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 		object.matrix = this.matrix.toArray();
 
+		// SkinnedMesh specific
+
+		if ( this.bindMode !== undefined ) object.bindMode = this.bindMode;
+		if ( this.bindMatrix !== undefined ) object.bindMatrix = this.bindMatrix.toArray();
+
```

Comment:
[`bindMatrixInverse` is calculated from `bindMatrix` in `SkinnedMesh.bind()`](https://github.com/mrdoob/three.js/blob/ec3a86668019835bbb4b729381f0dc5509417ea5/src/objects/SkinnedMesh.js#L113) so I didn't think we need to serialize `bindMatrixInverse`.
Do you think we should do that?